tests/Integration/Behaviour/Features/Context/Domain/Product/UpdateStatusFeatureContext.php

```diff
@@ -31,32 +31,57 @@
 use PHPUnit\Framework\Assert;
 use PrestaShop\PrestaShop\Core\Domain\Product\Command\UpdateProductStatusCommand;
 use PrestaShop\PrestaShop\Core\Domain\Product\Exception\ProductConstraintException;
+use PrestaShop\PrestaShop\Core\Domain\Shop\ValueObject\ShopConstraint;
 use Tests\Integration\Behaviour\Features\Transform\StringToBoolTransformContext;
 
 class UpdateStatusFeatureContext extends AbstractProductFeatureContext
 {
     /**
-     * @When /^I (enable|disable) product "(.*)"$/
+     * @When /^I (enable|disable) product "([^"]*)"$/
```

Comment:
Didn't it work fine previously?

Reply:
no it used to also match another scenario for multishop too. So I had to replace this regex to avoid matching `"`. Now it can match 2 references in same sentece like "shop1" and "product1". (previously it matched them like this "shop and product1")